### PR TITLE
Adds an IC issue button for admins

### PR DESCRIPTION
**Pull request by lzimann**
Makes the admins work a bit easier.

### CODE REVIEW

**Comment by kevinz000** [comment]
bad form.

File: code/modules/admin/topic.dm [hunk 1]
```diff
@@ -23,6 +23,20 @@
...
+		if(!C)
+			return
+
+		var/msg = "<font color='red' size='4'><b>- AdminHelp marked as IC issue! -</b></font><br>"
+		msg += "<font color='red'><b>Losing is part of the game!</b></font><br>"
+		msg += "<font color='red'>Your character will frequently die, sometimes without even a possibility of avoiding it. Events will often be out of your control. No matter how good or prepared you are, sometimes you just lose.</font>"
+
+		C << msg
+
+		message_admins("[key_name_admin(usr)] marked [C.key]'s as an IC issue.")
```
Comment (bgobandit) [inline code/modules/admin/topic.dm]: marked their what

**Comment by DoctorPork** [comment]
Fund it.

**Comment by Aloraydrel** [comment]
inb4 it gets spammed like the admin reject button and loses it's purpose

File: code/modules/admin/topic.dm (hunk 1 above)
Comment (lzimann) [inline code/modules/admin/topic.dm]: oops

**Comment by Niknakflak** [comment]
why do you care aloraydrel, you don't admin anyways 😆

**Comment by AnonymousNow** [comment]
I'm concerned that this will be a "ignore valid concerns" button.

**Comment by lzimann** [comment]
Admins can already use the REJECT button, ignore the ahelp itself or tell the player IC issue, it really doesn't change how things are already.

**Comment by Anonmare** [comment]
@AnonymousNow IC issue

File: code/modules/admin/topic.dm [hunk 2]
```diff
@@ -23,6 +23,20 @@
...
 		log_admin("[key_name(usr)] Rejected [C.key]'s admin help.")
 
+	else if(href_list["icissue"])
+		var/client/C = locate(href_list["icissue"])
+		if(!C)
+			return
+
+		var/msg = "<font color='red' size='4'><b>- AdminHelp marked as IC issue! -</b></font><br>"
+		msg += "<font color='red'><b>Losing is part of the game!</b></font><br>"
+		msg += "<font color='red'>Your character will frequently die, sometimes without even a possibility of avoiding it. Events will often be out of your control. No matter how good or prepared you are, sometimes you just lose.</font>"
```
Comment (PKPenguin321) [inline code/modules/admin/topic.dm]: preface with "Rule 10:

**Comment by ExcessiveUseOfCobblestone** [comment]
@KorPhaeron

File: code/modules/admin/topic.dm [hunk 3]
```diff
@@ -23,6 +23,20 @@
 		message_admins("[key_name_admin(usr)] Rejected [C.key]'s admin help. [C.key]'s Adminhelp verb has been returned to them.")
 		log_admin("[key_name(usr)] Rejected [C.key]'s admin help.")
 
+	else if(href_list["icissue"])
+		var/client/C = locate(href_list["icissue"])
```
Comment (optimumtact) [inline code/modules/admin/topic.dm]: is there really no more appropriately scoped list for this locate?

**Comment by Hatfish** [comment]
Can we add a link to the admin complaints forum as well?

File: code/modules/admin/topic.dm (hunk 3 above)
Comment (lzimann) [inline code/modules/admin/topic.dm]: Admin code so... I don't think so.

Comment (RemieRichards) [inline code/modules/admin/topic.dm]: could try ```in clients```, as we do have a clients list.

**Comment by ExcessiveUseOfCobblestone** [comment]
For those who didn't get my tagging of Kor,

we constantly meme about rule 10, hence why I tagged him on the "rule 10" pr.

**Pull request merged**